src/main/features/response/views/confirmation/part-admission-steps.njk

```diff
@@ -88,4 +88,19 @@
 
 {% endif %}
 
+{% if (claim.response.paymentIntention === undefined ) %}
+  <!-- States Paid -->
+  <h3 class="bold">{{ t('If {{ claimantName }} accepts your response', { claimantName: claimantName }) }}</h3>
+  <p>The claim will be settled.</p>
+  <h3 class="bold">{{ t('If {{ claimantName }} rejects your response', { claimantName: claimantName }) }}</h3>
+  {%  if claim.response.freeMediation === 'yes' %}
```

Comment:
can you please change yes to use the FreeMediationOption.YES value?

Reply:
Thanks @robertparkinson . Have sorted it out.